### PR TITLE
chore: remove custom logo

### PR DESCRIPTION
**Pull request by arshadali172**
## Problem

Closes https://github.com/opengovsg/FormSG/issues/787

## Solution

- Remove `form.customLogo` which was no longer being used
- Remove `form.hasheader` which was no longer being used
- Remove `form.logo` which was no longer being used
- Create script for deletion of `form.customLogo`

_Notes_
- Once `form.customLogo` has been deleted, such forms with either default to the config in `startPage.logo` or if `startPage.logo` is non-existent (i.e. for all new forms), default to the agency logo.

## Tests
- [ ] Create new form on email mode, should have default logo
- [ ] Create new form on storage mode, should have default logo
- [ ] Create new form on duplicate flow, should have default logo
- [ ] Create new form on template flow, should have default logo
- [ ] Edit form to have no logo
- [ ] Edit form to have custom logo

## Deploy Notes
- Script should be run before the code is merged.

### CODE REVIEW

File: scripts/20200106_remove-custom-logo/delete-custom-logo-key.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+*/
+
+// Check total forms count
+db.getCollection('forms').count() 
+
+// Check number of forms with customLogo key
+db.getCollection('forms').find({ customLogo: { $exists: true } }).count()
+
+// Check number of forms with startPage.logo key
+db.getCollection('forms').find({ "startPage.logo.state": { $exists: true } }).count()
```
Comment (liangyuanruo) [inline scripts/20200106_remove-custom-logo/delete-custom-logo-key.js:14]: can we also check how many forms have only the `customLogo` key but not `startPage.logo.state`?

**Review by liangyuanruo — APPROVED** [review]
Always happy to see a PR removing code. Generally LGTM but could I request expanding on the DB script so that we can be 100% sure that the `customLogo` key is safe to remove?

File: scripts/20200106_remove-custom-logo/delete-custom-logo-key.js (hunk 1 above)
Comment (arshadali172) [reply scripts/20200106_remove-custom-logo/delete-custom-logo-key.js:14]: Based on my queries yesterday:

**Total Number of forms:** 179888

**Forms with `startPage.logo.state` defined:** 139912

Breaks down into:
_DEFAULT_: 112081
_NONE_: 8297
_CUSTOM_: 19534

`customLogo` may exist for such forms too, but is currently ignored as `startPage.logo` is defined

**Forms with `startPage.logo.state` undefined:**  39977

Breaks down into:
`customLogo` undefined: 36359 => Currently defaults to agency logo
`customLogo` defined: 3618 => Currently defaults to `customLogo`, but once the key is removed, will default to agency logo

Comment (arshadali172) [reply scripts/20200106_remove-custom-logo/delete-custom-logo-key.js:14]: @liangyuanruo there are a sizeable number of forms (i.e. 36359) with neither `startPage.logo.state` nor `customLogo` as new forms are created without either key

**Comment by arshadali172** [comment]
> Always happy to see a PR removing code. Generally LGTM but could I request expanding on the DB script so that we can be 100% sure that the `customLogo` key is safe to remove?

how would you suggest I expand it? @liangyuanruo

File: scripts/20200106_remove-custom-logo/delete-custom-logo-key.js (hunk 1 above)
Comment (liangyuanruo) [reply scripts/20200106_remove-custom-logo/delete-custom-logo-key.js:14]: to clarify, this is because it's possible to have a form without a logo?

Comment (arshadali172) [reply scripts/20200106_remove-custom-logo/delete-custom-logo-key.js:14]: *Old Code*
- If `startPage.logo.state` present, will use state defined therein
- If `startPage.logo.state` absent, will use state defined in `customLogo`. Note that `undefined` state for `customLogo` means that the default agency logo is to be shown
- New forms are created with `startPage.logo.state` absent and `customLogo` `undefined` and hence show the default agency logo

*New Code*
- If `startPage.logo.state` present, will use state defined therein
- Will ignore `customLogo`, although `customLogo` should have been removed from `form` documents
- If `startPage.logo.state` absent, default agency logo will be shown
- New forms are created with `startPage.logo.state` absent and hence show the default agency logo

Comment (arshadali172) [reply scripts/20200106_remove-custom-logo/delete-custom-logo-key.js:14]: does that make sense @liangyuanruo

**Pull request merged**